openslides_backend/migrations/migrations/0038_remove_user_default_projector.py

```diff
@@ -0,0 +1,10 @@
+from datastore.migrations import RemoveFieldsMigration
+
+
+class Migration(RemoveFieldsMigration):
+    """
+    This migration removes the field `meeting/default_projector_$user_ids`.
```

Comment:
Ah, I forgot: `user` must also be removed from `meeting/default_projector_$_ids`, if present. Please extend this migration by overwriting the `migrate` method.

Reply:
Done.